provisioning/device/src/Microsoft.Azure.Devices.Provisioning.Client.csproj

```diff
@@ -20,7 +20,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <VersionPrefix>1.0.1</VersionPrefix>
+    <VersionPrefix>1.0.2</VersionPrefix>
```

Comment:
Were there any ProvisioningClient versions? Do we need to release this?

Reply:
Removing unnecessary CodeAnalysis.FxCopAnalyzers IOperation declaration changed the csproj.  I didn't look at every change in details.  There were some changes in "shared" in other PR.  If you are sure they do not need to be updated/release, we could revert.

Reply:
Discussed offline: OK to leave as is. The FXCop change should not affect the behavior but there were changes to shared that might have been functional.